scan/src/components/modal/submitTx/PreviewJsonStep.re

```diff
@@ -111,140 +53,149 @@ let make = (~rawTx, ~onBack, ~account: AccountContext.t) => {
+                               dispatchModal(EnableExit);
+                               Js.Promise.resolve();
+                             }
+                           )
+                        |> Js.Promise.catch(err => {
+                             switch (Js.Json.stringifyAny(err)) {
+                             | Some(errorValue) => setState(_ => Error(errorValue))
+                             | None => setState(_ => Error("Can not stringify error"))
+                             };
                              dispatchModal(EnableExit);
                              Js.Promise.resolve();
-                           }
-                         )
-                      |> Js.Promise.catch(err => {
-                           switch (Js.Json.stringifyAny(err)) {
-                           | Some(errorValue) => setState(_ => Error(errorValue))
-                           | None => setState(_ => Error("Can not stringify error"))
-                           };
-                           dispatchModal(EnableExit);
-                           Js.Promise.resolve();
-                         }),
-                    );
+                           }),
+                      );
 
-                    Promise.ret();
-                  })
-               |> Js.Promise.catch(_ => {
-                    setState(_ => Error("Failed to sign message"));
-                    dispatchModal(EnableExit);
-                    Promise.ret();
-                  });
-             ();
-           }}>
-           <Text value="Broadcast" weight=Text.Bold size=Text.Md color=Colors.white />
+                      Promise.ret();
+                    })
+                 |> Js.Promise.catch(_ => {
+                      setState(_ => Error("Failed to sign message"));
+                      dispatchModal(EnableExit);
+                      Promise.ret();
+                    });
+               ();
+             }}>
+             <Text value="Broadcast" weight=Text.Semibold size=Text.Lg color=Colors.white />
+           </Button>
          </div>
+         <a className=Styles.btnBack onClick=onBack>
+           <Text value="Back" weight=Text.Semibold size=Text.Lg color=Colors.gray7 />
+         </a>
        </div>
      | Success(txHash) =>
-       <div className=Styles.resultContainer>
-         <div className=Styles.rFlex id="successMsgContainer">
-           <img src=Images.success2 className=Styles.resultIcon />
-           <HSpacing size=Spacing.md />
-           <Text value="Broadcast Transaction Success" weight=Text.Semibold />
+       <div
+         className={Css.merge([
+           CssHelper.flexBox(~direction=`column, ~justify=`center, ()),
+           Styles.resultContainer,
+         ])}>
+         <img src=Images.success className=Styles.resultIcon />
+         <div id="successMsgContainer" className={CssHelper.mb(~size=16, ())}>
+           <Text value="Broadcast Transaction Success" size=Text.Lg block=true align=Text.Center />
```

Comment:
Please check that the message not match with the design and don't have to be all caps.